trail/query/views.py

```diff
@@ -1,6 +1,42 @@
-from django.http import HttpResponse
+from django import forms
+# from django.forms import ModelForm
+from django.apps import apps
+from django.shortcuts import render
+
+
+Metadata = apps.get_model('upload', 'Metadata')
+# Needs to think about adding SELECT DISTINCT etc.
+
+
+class MetadataForm(forms.Form):
+    unique_instrument = ["DECam", "ff09", "text", "option"]
+    u_instrlist = list((name, name) for name in unique_instrument)
+    instrument = forms.CharField(max_length=20, widget=forms.Select(choices=u_instrlist))
+    telescope = forms.CharField(max_length=20, required=False)
+
+    processor_name = forms.CharField(max_length=20, required=False)
 
 
-# not used
 def index(request):
-    return HttpResponse("Wouldn't it be nice if someone wrote a query tool here?")
+    """Index renders the form when root url is visited, bypassing slower checks
+    required fo rendering of the results url, where rendering of the table and
+    checking for results are performed.
+
+    It is assumed the request type is GET.
+    """
+    form = MetadataForm()
+    return render(request, "query.html", {"queryform": form, "render_table": False})
+
+
+def print_results(request):
+    """Renders the results url, which is a placeolder copy of the root url of
```

Comment:
Typo - "placeholder" not "placeolder"

Reply:
I didn't type this part but have fixed it.